Description: don't let "reopen-log" command rotate logs
 Make sure the "ejabberdctl reopen-log" command really just reopens log
 files and doesn't also rename them.  This allows us to use it as a
 "postrotate" command in our logrotate configuration for ejabberd.
Author: Holger Weiss <holger@zedat.fu-berlin.de>

Index: ejabberd/src/ejabberd_logger.erl
===================================================================
--- ejabberd.orig/src/ejabberd_logger.erl
+++ ejabberd/src/ejabberd_logger.erl
@@ -129,13 +129,7 @@ start() ->
     ok.
 
 reopen_log() ->
-    lager_crash_log ! rotate,
-    lists:foreach(
-      fun({lager_file_backend, File}) ->
-              whereis(lager_event) ! {rotate, File};
-         (_) ->
-              ok
-      end, gen_event:which_handlers(lager_event)).
+    ok.
 
 get() ->
     case lager:get_loglevel(lager_console_backend) of
